mintapi/api.py

```diff
@@ -649,8 +649,8 @@ def __build_payload(
         return request(
             date_filter=DateFilter(
                 date_filter=date_filter,
-                start_date=convert_mmddyy_to_datetime(start_date),
-                end_date=convert_mmddyy_to_datetime(end_date),
+                start_date=convert_mmddyy_to_datetime(start_date).strftime("%Y-%m-%d"),
```

Comment:
I'd prefer if we just update the method so we're ways returning the date in its required format.